src/components/edit/RecentChangeHistory.tsx

```diff
@@ -51,7 +52,7 @@ const ChangesetRow = ({ changeset }: ChangsetRowProps): JSX.Element => {
 }
 
 const ClimbChange = ({ changeId, fullDocument, updateDescription, dbOp }: ChangeType): JSX.Element | null => {
-  if ((fullDocument as ClimbType)?.name == null) {
+  if (fullDocument.__typeName !== DocumentTypeName.Climb) {
```

Comment:
More robust to use __typename than unique document fields: https://www.apollographql.com/docs/apollo-server/schema/unions-interfaces/#querying-a-union